stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -49,27 +52,36 @@ function externAbsolute(handle path) returns handle|Error =
     class: "org.ballerinalang.stdlib.filepath.nativeimpl.FilePathUtils"
 } external;
 
-# Returns path separator of underline operating system.
+# Returns the path separator of the underlying operating system.
+# ```ballerina
+#  string pathSeparator = filepath:getPathSeparator();
+# ```
 #
-# + return - String value of path separator
+# + return - String value of the path separator
 public function getPathSeparator() returns string {
     return pathSeparator;
 }
 
-# Returns path list separator of underline operating system.
+# Returns the path list separator of the underlying operating system.
```

Comment:
What's a path list separator? Is that a standard/common term? If not, shall we define what we mean by a "path list"?

Reply:
Corrected.